modules/augment/include/opencv2/augment/flip.hpp

```diff
@@ -35,6 +35,29 @@ namespace augment {
     };
 
 
+
+    class CV_EXPORTS_W FlipVertical : public Transform
+    {
+    public:
+        /* @brief Constructor
+           @param _probability probability that this transformation will be applied to each image
+        */
+        CV_WRAP FlipVertical(const Scalar& _probability);
+
+        /* @brief Apply the vertical flipping to a single image
+           @param _src Input image to be flipped
+           @param _dst Output (flipped) image
+        */
+        CV_WRAP void image(InputArray _src, OutputArray _dst);
```

Comment:
The name of function should describe what it does. The trasformations are applied. They are applied to images or to rectangles or something else. So I'd like to rename it to `apply`.

Reply:
I think on the higher level we can't name them all apply, because we would need to recieve Mat in all of them (recieving image or np.array of bboxes or points), so naming them with the data type they work on would be more consistent on the long run.

should I rename them to applyImage() for example?

Reply:
plus like bbox function takes (Mat image, Mat boxes) and points function take (Mat image, Mat points), so renaming them to different names while naming the single point and bbox to just apply would make a very confusing api

Reply:
You do need to define boxes by `cv::Mat`. There is [cv::Rect](https://docs.opencv.org/master/dc/d84/group__core__basic.html#ga11d95de507098e90bad732b9345402e8)

Reply:
I need list of boxes per image so it would have be 2d Mat, plus all datasets load bboxes in 2d np.arrays or pandas dataframe which could be easily transformed to np.array